extensions/amp-base-carousel/0.1/amp-base-carousel.js

```diff
@@ -449,7 +449,7 @@ class AmpCarousel extends AMP.BaseElement {
       'goToSlide',
       (actionInvocation) => {
         const {args, trust} = actionInvocation;
-        this.carousel_.goToSlide(args['index'] || -1, {
+        this.carousel_.goToSlide(args['index'] ?? -1, {
```

Comment:
Can you explain what the behavior was before and what the behavior is now with the fix?

Reply:
Sure, before when a user created an amp-base-carousel and set one of the buttons to `goToSlide(index=0)` the carousel wouldn't move. Expected behavior was to go to the first slide.

Now, when a user sets `goToSlide(index=0)` on a carousel button, it will go to the first slide.

Reply:
Awesome, good to document the root cause of the issue.